pkg/ccl/backupccl/backup_test.go

```diff
@@ -6735,7 +6736,7 @@ func TestRestoreErrorPropagates(t *testing.T) {
 	defer dirCleanupFn()
 	params := base.TestClusterArgs{}
 	params.ServerArgs.ExternalIODir = dir
-	jobsTableKey := keys.SystemSQLCodec.TablePrefix(keys.JobsTableID)
+	jobsTableKey := keys.SystemSQLCodec.TablePrefix(uint32(systemschema.JobsTable.GetID()))
```

Comment:
Naturally this change (and the others like it) are motivated by wanting to make some of these system tables' IDs dynamically defined instead of hard-coded. Still, in the meantime, we might as well refer to them via `systemschema` when possible. There's a very minor benefit of type safety, of course, and I'd rather we refer to the `keys` constant only in cases where we actually _want_ the table ID to be hard-coded for performance reasons, typically in cases where we want to build a `roachpb.Key` without having to do lookups.